dev/ApplicationData/ApplicationData.idl

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include <TerminalVelocityFeatures-ApplicationData.h>
+
+namespace Microsoft.Windows.Storage
+{
+    [contractversion(2)]
```

Comment:
Because `GetForUnpackaged()` isn't ready at this time so rather than hold up the rest (Perfect is Enemy of the Good, etc) this is marked v2 and experimental to be addressed later